services/productService.js

```diff
@@ -0,0 +1,14 @@
+const productDao = require("../models/productDao");
+
+const getProducts = async (req) => {
+  return await productDao.getProducts(req);
+};
```

Comment:
```suggestion
const getProducts = async (req) => {
  return productDao.getProducts(req);
};
```
다음의 코드 뒤에 동기적인 실행을 기다리고 있는 로직이 없기 때문에 await를 쓰지 않으셔도 됩니다.

Reply:
네 수정했습니다 !